mobile/native/src/trade/order/handler.rs

```diff
@@ -21,10 +24,31 @@ pub async fn submit_order(order: OrderTrade) -> Result<()> {
 
     event::publish(&EventInternal::OrderUpdateNotification(order));
 
+    // in a day's time
+    let maturity_time = (SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .unwrap()
+        .as_secs()
+        + 86_400) as u32;
+
+    // TODO: fetch open (market) price from e.g. bitmex
+    let open_price: f64 = 55_000.0;
+
     // TODO: remove this and use the orderbook event to trigger the trade!
     let dummy_trade_params = TradeParams {
-        taker_node_pubkey: get_node_pubkey(),
-        contract_input: ContractInput {},
+        taker_node_pubkey: ln_dlc::get_node_info()?.pubkey,
+        contract_input: ContractInput {
+            maturity_time,
+            taker_margin: calculations::calculate_margin(
+                open_price,
+                order.quantity,
+                order.leverage,
+            ),
+            // TODO: What would we choose as leverage for the coordinator? is it the same leverage
```

Comment:
I think this TODO is clear now: 
We always want a leverage of 1 for the coordinator.